core/templates/components/forms/custom-forms-directives/thumbnail-uploader.component.ts

```diff
@@ -34,37 +34,37 @@ import { EditThumbnailModalComponent } from './edit-thumbnail-modal.component';
   templateUrl: './thumbnail-uploader.component.html'
 })
 export class ThumbnailUploaderComponent implements OnInit, OnChanges {
-  @Input() disabled: boolean;
-  @Input() useLocalStorage: boolean;
-  @Input() allowedBgColors: string[];
-  @Input() aspectRatio: string;
-  @Input() bgColor: string;
-  @Input() filename: string;
-  @Input() previewDescription: string;
-  @Input() previewDescriptionBgColor: string;
-  @Input() previewFooter: string;
-  @Input() previewTitle: string;
-  @Output() updateBgColor: EventEmitter<string> = new EventEmitter() ;
+  @Input() disabled!: boolean;
```

Comment:
Please add a comment explaining the usage of `!`, you can find this comment in other already strictly typed files.

Reply:
Added the comments